src/include/test/unit_utils.cc

```diff
@@ -33,6 +33,14 @@
 #include "config.h"
 #include "utils/utils.h"
 
+namespace {
```

Comment:
Do we need this any longer?  If we are still using `stdx::filesystem::path`, then maybe it should go into that header?  I think with the approach we are taking now, we shouldn't need `stdx::filesystem::path`?

Reply:
I did commit this by mistake, will remove